MinecraftClient/Program.cs

```diff
@@ -401,12 +399,13 @@ private static void InitializeClient()
         /// Disconnect the current client from the server and restart it
         /// </summary>
         /// <param name="delaySeconds">Optional delay, in seconds, before restarting</param>
-        public static void Restart(int delaySeconds = 0)
+        public static void  Restart(int delaySeconds = 0)

```

Comment:
```suggestion
        public static void Restart(int delaySeconds = 0)
```

Reply:
I thought I removed this space, but somehow this wasn't a problem in the first place? 
That's what I get for writing code at 3 am 🥴